core/arch/arm/include/kernel/embedded_sp.h

```diff
@@ -0,0 +1,16 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2020, Arm Limited.
+ */
+#ifndef KERNEL_EMBEDDED_SP_H
+#define KERNEL_EMBEDDED_SP_H
+
+#include <kernel/embedded_ts.h>
+#include <stdint.h>
+#include <tee_api_types.h>
+
+#define for_each_embedded_sp(_sp) \
+	SCATTERED_ARRAY_FOREACH(_sp, embedded_sps, struct embedded_ts)
+
+#endif /* KERNEL_EMBEDDED_SP_H */
```

Comment:
minor: thie change belongs to commit "core: create embedded_ts".

Reply:
embedded_sp.h is only created in core: add embedded SPs

Reply:
current commit "core/arch/arm/include/kernel/embedded_ts.h" creates **embedded_sp.h**.
then s/`KERNEL_EARLY_SP_H`/`KERNEL_EMBEDDED_SP_H`/ is currently fixed in later commit "core: add embedded SPs".